optimum/exporters/onnx/convert.py

```diff
@@ -112,7 +112,8 @@ def export_pytorch(
             model.to(device)
             dummy_inputs = tree_map(lambda value: value.to(device), dummy_inputs)
         check_dummy_inputs_are_allowed(model, dummy_inputs)
-        inputs = config.ordered_inputs(model)
+        # inputs = config.ordered_inputs(model)
```

Comment:
The  will always return the same name of the input as was present in the original model. However, in some cases we may want to change the input name in the generated `ONNX` model.

For example: For the following config [SpeechSeq2SeqDecoderOnnxConfig](https://github.com/huggingface/optimum/pull/420/files#diff-cf7d52b6c0ee70bc45f40c2c40ac99b49dbf02f2fdb0a141e9483f764126e05eR709), it makes sense to have a different input names than the dummy inputs provided to the model for export.

Currently I have just added a temporary change to test the end-to-end pipeline. @michaelbenayoun could you suggest a permanent fix? One could be to revert back to using `OrderedDict` for `inputs`?

Reply:
As discussed would create a mapping between the input names to solve the issue

Reply:
Updated in [af7ddd3](https://github.com/huggingface/optimum/pull/420/commits/af7ddd37c11033b6bcd4d654c3604f7737f8767d)